### ISSUE TITLE
Flex Layout not wrapped text correctly for Label

### ISSUE THREAD

**Issue by Chozarajan** [xamarin/Xamarin.Forms#5311]
### Description
The Label text was not shown fully in Flex Layout. The only first part of the text was shown. Check the attached screenshot for more details.  

### Steps to Reproduce
Code :
```
<FlexLayout
                        HorizontalOptions="Start" 
                        VerticalOptions="Center" 
                        Direction="Row" 
                        Wrap="Wrap"
                        JustifyContent="Start"
                        AlignContent="Start" 
                        AlignItems="Start">
                <Label Margin="8,0,8,0" Text="Elvis A. Presley" LineBreakMode="NoWrap"/>
                <Label Margin="8,0,8,0" Text="John Doee" LineBreakMode="NoWrap"/>
                <Label Margin="8,0,8,0" Text="Ryan Gig" LineBreakMode="NoWrap"/>
                <Label Margin="8,0,8,0" Text="Rathana Kumar" LineBreakMode="NoWrap"/>
                </FlexLayout>
```

1.  Run the above code
2. The Label text was not shown fully

### Expected Behavior
Need to display the full text in the Label

### Actual Behavior
Only the first part of the Label text only shown 

### Basic Information

- Version with issue: 7.1
- Last known good version: - 
- IDE: VS 17 Mac
- Platform Target Frameworks: 
  - iOS:  -
  - Android: 7.1
  - UWP:  -
- Android Support Library Version: 
- Nuget Packages: -
- Affected Devices: Currently checked in Moto G5 Plus.

### Screenshots
![image](https://user-images.githubusercontent.com/42271947/53014785-7d513900-346f-11e9-95ee-c1c1081886ac.png)

### Reproduction Link
Use the below code 

```
<FlexLayout 
                        HorizontalOptions="Start" 
                        VerticalOptions="Center" 
                        Direction="Row" 
                        Wrap="Wrap"
                        JustifyContent="Start"
                        AlignContent="Start" 
                        AlignItems="Start">
                <Label Margin="8,0,8,0" Text="Elvis A. Presley" LineBreakMode="NoWrap"/>
                <Label Margin="8,0,8,0" Text="John Doee" LineBreakMode="NoWrap"/>
                <Label Margin="8,0,8,0" Text="Ryan Gig" LineBreakMode="NoWrap"/>
                <Label Margin="8,0,8,0" Text="Rathana Kumar" LineBreakMode="NoWrap"/>
                </FlexLayout>
```

**Comment by pauldipietro** [comment]
I placed that bit of XAML into a Shell template app and the full names display. Do you experience this on all devices, or only specific ones such as the one mentioned? Which version of Forms are you using?

**Comment by Eswaran17392** [comment]
Yes, this issue occurs in all the devices and the devices which we checked are MOTO G4, Samsung S8 and  MOTO G5 devices. We used the latest Xamarin forms version 3.5.0.169047.

**Comment by Chozarajan** [comment]
I have checked with other devices if we **adjust the Margin** values the same issue came in Lenovo K4, Moto G4, Moto E2 devices.

**Comment by Chozarajan** [comment]
> Yes, this issue occurs in all the devices and the devices which we checked are MOTO G4, Samsung S8 and MOTO G5 devices. We used the latest Xamarin forms version 3.5.0.169047.

**Comment by Adam-Langley** [comment]
Workaround - replace Labels with Buttons. Nasty, but fixes the issue for me.

**Comment by gauravsk1996** [comment]
I am also getting the exact same issue, but my label is inside the Frame component
<img width="307" alt="image" src="https://user-images.githubusercontent.com/24958672/166872939-4376be41-7203-451f-9c6f-766f95a405e1.png">
Actual text is "Awaiting Delayed Start" and "SN : 20754527"

**Comment by gauravsk1996** [comment]
@Adam-Langley I replaced the label (in my case frame->label) with the button but it still doesn't work right for me.
<img width="304" alt="image" src="https://user-images.githubusercontent.com/24958672/172351218-ed489f1a-ae98-4f36-a8c4-472a9e9da08c.png">
Style for button is :
Margin - 5, padding - 5,0, IsEnabled - False, CornerRadius - 4, HorizontalOptions - Fill, ContentLayout - Left,5, HeightRequest - 25.
Do you have any suggestions on how I can avoid text going to the second line?
